packages/manager/modules/pci/src/projects/project/notebooks/components/configuration-command/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,5 @@
+{
+  "pci_notebook_configuration_command_description": "With the following command, you can create the same notebookusing our ovhai CLI:",
+  "pci_notebook_configuration_command_help": "For more information, you can read the documentation about ovhai CLI",
+  "pci_notebook_configuration_command_error": "Une erreur s'est produite lors de la tentative de récupération de la commande équivalente"
```

Comment:
Missing content I think

Reply:
Yeah the content is missing as of now. We have a separate task to track the contents...